libraries/stdlib/genglsl/lib/mx_noise.glsl

```diff
@@ -172,8 +172,8 @@ uint mx_hash_int(int x, int y)
 uint mx_hash_int(int x, int y, int z)
 {
     uint a, b, c;
-    uint len = 3;
-    a = b = c = 0xdeadbeef + (len << 2) + 13;
+    uint len = 3u;
+    a = b = c = uint(0xdeadbeef) + (len << 2u) + 13u;
```

Comment:
Maximum uint value is 0xffffffff so 0xdeadbeef lies within the range and won't change behaviour.